name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/BarclaysPDFExtractor.java

```diff
@@ -39,6 +39,30 @@ private void addCreditcardStatementTransaction()
                                         .assign((ctx, v) -> ctx.put("currency", asCurrencyCode(v.get("currency")))));
 
         this.addDocumentTyp(type);
+        
+        Block depositBlock = new Block("^[\\d]{2}\\.[\\d]{2}\\.[\\d]{4} [\\d]{2}\\.[\\d]{2}\\.[\\d]{4}.* [\\.,\\d]+\\+$");
+        type.addBlock(depositBlock);
+        depositBlock.set(new Transaction<AccountTransaction>()
+
+                        .subject(() -> {
+                            AccountTransaction accountTransaction = new AccountTransaction();
+                            accountTransaction.setType(AccountTransaction.Type.DEPOSIT);
+                            return accountTransaction;
+                        })
+
+                        .section("date", "note", "amount") //
+                        .documentContext("currency") //
+                        .match("^[\\d]{2}\\.[\\d]{2}\\.[\\d]{4} (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) " //
+                                        + "(?<note>.{1,36}).+ +" //
+                                        + "(?<amount>[\\.,\\d]+)\\+$") //
+                        .assign((t, v) -> {
+                            t.setDateTime(asDate(v.get("date")));
+                            t.setAmount(asAmount(v.get("amount")));
+                            t.setCurrencyCode(v.get("currency"));
+                            t.setNote(v.get("note").trim());
```

Comment:
```suggestion
                            t.setNote(trim(v.get("note")));
```

Reply:
What's the difference?

Reply:
See [here ](https://github.com/portfolio-performance/portfolio/blob/08f27032ef0fd6655c1fe60b7c5437582a3e42c4/name.abuchen.portfolio/src/name/abuchen/portfolio/util/TextUtil.java#L89:L125)